.github/workflows/wheels.yml

```diff
@@ -129,19 +129,17 @@ jobs:
       fail-fast: false
 
       matrix:
-        #os: [macos-10.15, windows-latest]
-        #os: [macos-10.15, macOS-M1]
-        os: [macos-10.15]
+        os: [macos-latest, windows-latest]
         python_version: ["2.7", "3.6", "3.7", "3.8", "3.9", "3.10", "pypy-3.7-v7.3.3", "pypy-3.8-v7.3.7"]
```

Comment:
Please add "3.11" ?

Reply:
Yes, I was planning on doing that in a separate PR, because the Python 3.11 job might discover some bugs, but I added it now!